osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -89,6 +91,8 @@ public override bool PauseImports
             {
                 PostNotification = obj => PostNotification?.Invoke(obj)
             };
+
+            execution = new Queue<Task>();
```

Comment:
so nothing dequeues anything from this ever and this is allowed to grow ad infinitum? no, sorry, that is unacceptable.